media.graphql

```diff
@@ -0,0 +1,185 @@
+# This GraphQL schema defines the META Media types and queries.
+#
+# It consists of the following main entities:
+#
+# MusicPerformer
+# MusicComposer
+# RecordLabel
+# MusicPublisher
+# MusicRecording
+# MusicWork
+# MusicRelease
+# MusicProduct
+#
+# Each entity has a set of widely used identifiers which are used to uniquely
+# identify them:
+#
+# ISNI - International Standard Name Identifier
+# IPI  - Interested Parties Information
+# DPID - DDEX Party Identifier
+# ISRC - International Standard Recording Code
+# ISWC - International Standard Musical Work Code
+# GRid - Global Release Identifier
+# UPC  - Universal Product Code
+#
```

Comment:
What about EAN ?
https://en.wikipedia.org/wiki/International_Article_Number
As ISRC number which is used to identify an individual recording a UPC or EAN number is used to identify a group of recordings or a physical product.
CWR - use EAN (and not UPC) to link a recording.

Reply:
Is there a benefit of one over the other? I hadn't heard of EAN tbh, wheras UPC is commonly known. To be clear i have no reservations either way.

Reply:
Sure, feel free to add that in.

Reply:
Added in 1a0a39b.